### PR TITLE
[PhpUnitBridge] ExcludeList usage for PHPUnit 9.4

### PR DESCRIPTION
**Pull request by gennadigennadigennadi**
| Q             | A
| ------------- | ---
| Branch?       |  4.4 
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Tickets       | #37344 
| License       | MIT

Added: ExcludeList usage for PHPUnit >= 9.3

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
Great thanks. Can you please target branch 3.4?

**Comment by gennadigennadigennadi** [comment]
The fix (CommandForV9) for PHPUnit 9 is missing inside the 3.4 Branch.
I have not checked if 3.4 is capable of running PHPUnit 9, but I do have some doubts.

Are the fixes for PHPUnit 9 not required in Symfony 3.4? At least I have not not seen them inside the 3.4 Branch.

**Comment by nicolas-grekas** [comment]
> The fix (CommandForV9) for PHPUnit 9 is missing inside the 3.4 Branch

Oh, OK for 4.4 then!

File: src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php [hunk 1]
```diff
@@ -254,7 +254,11 @@
 if (!class_exists('SymfonyBlacklistPhpunit', false)) {
     class SymfonyBlacklistPhpunit {}
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php:255]: can you rename this class also please?
SymfonyExcludelistPhpunit

Comment (nicolas-grekas) [reply src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php]: (fixed)

**Comment by nicolas-grekas** [comment]
Thank you @gennadigennadigennadi.

**Pull request merged**